edit-post/editor.js

```diff
@@ -9,16 +9,26 @@ import { EditorProvider, ErrorBoundary } from '@wordpress/editor';
  */
 import Layout from './components/layout';
 
-function Editor( { settings, hasFixedToolbar, onError, ...props } ) {
+function Editor( { settings, hasFixedToolbar, post, defaultPost, onError, ...props } ) {
+	if ( ! post ) {
+		return null;
+	}
+
+	const editorSettings = {
+		...settings,
+		hasFixedToolbar,
+	};
+
 	return (
-		<EditorProvider settings={ { ...settings, hasFixedToolbar } } { ...props }>
+		<EditorProvider settings={ editorSettings } post={ { ...post, ...defaultPost } } { ...props }>
```

Comment:
Shouldn't the spread be the other way around? `{ ...defaultPost, ...post }`

Reply:
>Shouldn't the spread be the other way around? { ...defaultPost, ...post }

Still wondering about this note.

Reply:
Okay, I see this may be needed because auto-draft title replacement is meant to take precedent over the default title. I wonder if we could...

- Avoid needing this override in the first place (set auto-draft title as it's created?)
- Rename the variable to reflect that it's an override, not a set of fallback values
- Comment to reflect the intended usage

Reply:
I think it's also needed for the new post, since auto-draft is created with a "(Auto-draft)" title and we don't want to show this in the editor, so we needed to be able to override, though I think ideally we just... don't assign that title 😄